openpype/pipeline/__init__.py

```diff
@@ -82,5 +98,19 @@
     "PublishValidationError",
     "PublishXmlValidationError",
     "KnownPublishError",
-    "OpenPypePyblishPluginMixin"
+    "OpenPypePyblishPluginMixin",
+
+    # --- Plugins ---
```

Comment:
Might this be better labeled actions? Because above in load there are also "plugins"?

```suggestion
    # --- Actions ---
```